src/EditorFeatures/Core/Implementation/InlineRename/AbstractEditorInlineRenameService.cs

```diff
@@ -96,7 +96,7 @@ public async Task<IInlineRenameInfo> GetRenameInfoAsync(Document document, int p
             if (symbol.IsImplicitlyDeclared &&
                 symbol.Kind != SymbolKind.Local &&
                 !(symbol.Kind == SymbolKind.Parameter &&
-                  symbol.ContainingSymbol.Kind == SymbolKind.Method &&
+                  symbol.ContainingSymbol?.Kind == SymbolKind.Method &&
```

Comment:
💭 Do we not have an `IsKind(ISymbol?, SymbolKind)` extension method?

Reply:
good call